src/server/common/templates/partials/get-help.njk

```diff
@@ -1,7 +1,7 @@
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
         <h3 class="govuk-heading-s govuk-!-margin-top-2">{{supportSection.title}}</h3>
-        <p class="govuk-body-s govuk-!-margin-bottom-1">{{supportSection.description}}</p>
-        <a class="govuk-link govuk-body-s" href="mailto:planthealth.info@apha.gov.uk?subject=Plant Health Information Service">planthealth.info@apha.gov.uk</a>
+        <p class="govuk-body govuk-!-margin-bottom-1">{{supportSection.description}}</p>
+        <a class="govuk-link govuk-body" href="mailto:planthealth.info@apha.gov.uk?subject=Plant Health Information Service">planthealth.info@apha.gov.uk</a>
```

Comment:
can this href value be also picked up from a config file

Reply:
@manabshy had a discussion with @SajuJ regarding this since href values does not change we are not adding it to config file